### PR TITLE
draft release 099

### PR DESCRIPTION
**Pull request by bpathakota**
## What issues does this PR fix or reference?

- Fixes:
OR
- Relates to:

> **NOTE:** `Fixes` will close the ticket after merging. `Relates to:` will leave the ticket open after merging.

### Merge requirements satisfied

> For more information, visit the [Contributing Guide for Documentation](https://wiki.dendron.so/notes/b58801fc-43a9-4d42-a58b-eabc3e8538cb/)

- [ ] Check rendered output to ensure formatting is correct for renderings of wikilinks, note refs, tables, and images. `Dendron: Show Preview` can be used in your workspace to confirm the page renders as expected. Sometimes, `Dendron: Reload Index` needs to be ran if certain wikilinks aren't working as expected. (ignore this if contribution is made via the `Edit this page on GitHub` button from the published site)
- [ ] If this PR includes any refactoring (renaming notes, renaming headers, etc.), make sure that these changes were done via [Dendron Refactoring Commands](https://wiki.dendron.so/notes/srajljj10V2dl19nCSFiC/). This will ensure that any other impacted areas of the documentation, that link to or embed modified notes (via note references), are automatically updated.

Verify GitHub Actions tests are passing, which can be seen in the `Checks` tab of the PR:

- [ ] `URL validator` GitHub Action passes
- [ ] `Dendron site build` GitHub Action passes

> **NOTE:** If you are a new contributor, a maintainer will need to provide approval for GitHub Actions to run on your PRs.

### CODE REVIEW

File: vault/changelog.md [hunk 1]
```diff
@@ -6,6 +6,24 @@ updated: 1653971733422
 created: 1601508213606
 nav_order: 2
 ---
+
+## 0.99.0
+
+### Features
+- feat(structure): add set task status commands (#3033) @kaan ([[docs|dendron://dendron.dendron-site/dendron.ref.commands#set-task-status]])
```
Comment (kevinslin) [inline vault/changelog.md:13]: `commands` should be singular (`command`)

File: vault/changelog.md [hunk 2]
```diff
@@ -6,6 +6,24 @@ updated: 1653971733422
...
+- feat(structure): add complete task commands (#3033) @kaan ([[docs|dendron://dendron.dendron-site/dendron.ref.commands#complete-task]])
```
Comment (kevinslin) [inline vault/changelog.md:14]: same as above

File: vault/changelog.release.2022-06-14.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+## Community
+
+### General Updates
+
+#### Public Issues
+    The public data view of frequently asked questions and how to fix them is live on our [FAQ Page](https://wiki.dendron.so/notes/683740e3-70ce-4a47-a1f4-1f140e80b558/) under 'Public Issues'. We will continuously update this list with issues we seen on discord and when working with community members. Check it out and please give us feedback in the [[#feedback|dendron://dendron.dendron-site/community.discord.channels#feedback]] channel or at support@dendron.so.
+
+#### Greenhouse Series
+
+    ![[community.events.greenhouse#summary,1:#*]]
```
Comment (kevinslin) [inline vault/changelog.release.2022-06-14.md:40]: looks like theres a tab/space character here? shouldn't be there

File: vault/changelog.md (hunk 1 above)
Comment (bpathakota) [reply vault/changelog.md:13]: ack

File: vault/changelog.release.2022-06-14.md (hunk 3 above)
Comment (bpathakota) [reply vault/changelog.release.2022-06-14.md]: updated

**Review by kevinslin — APPROVED** [review]

**Pull request merged**